config/memorysize.go

```diff
@@ -113,7 +113,7 @@ func (m MemorySize) MarshalText() ([]byte, error) {
 func (m *MemorySize) UnmarshalText(text []byte) error {
 	txt := string(text)
 
-	r := regexp.MustCompile(`^\s*(?P<number>[0-9.]+)(?P<unit>[a-zA-Z]*)\s*$`)
+	r := regexp.MustCompile(`^\s*(?P<number>[0-9\._]+)(?P<unit>[a-zA-Z]*)\s*$`)
```

Comment:
Most metacharacters inside character classes (square brackets) don't apply, so you don't need to escape the `.`, so the backslash before is harmless but unnecessary. Please remove it as we don't use it elsewhere. The missing _ is a good catch, though.